docs/reference/release-notes/7.6.asciidoc

```diff
@@ -1,3 +1,74 @@
+[[release-notes-7.6.1]]
+== {es} version 7.6.1
+
+coming[7.6.1]
```

Comment:
Can we remove `coming[7.6.1]` before merging this today?

Reply:
Removed with 678ad77ed63.